workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/UtilityFunctionsMapper.java

```diff
@@ -67,6 +76,49 @@ public static String escape(String s) {
     return new String(JsonStringEncoder.getInstance().quoteAsString(s));
   }
 
+  public static String encodeRawQuery(String url) throws UnsupportedEncodingException {
+    return encodeRawQuery(url, StandardCharsets.UTF_8.name());
+  }
+
+  public static String encodeRawQuery(String url, String encoding)
```

Comment:
do we want to expose that? given all workflows are probably expected to be utf-8 ?